drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -67,6 +67,9 @@ function DragAndDropEditBlock(runtime, element, params) {
 
                     // Hide settings that are specific to assessment mode
                     _fn.build.$el.feedback.form.find('.problem-mode').trigger('change');
+
+                    // Set focus on first input field.
+                    $element.find('input:first').select();
```

Comment:
@cptvitamin can weigh in on this during the a11y review. I do wonder which is better-- moving focus to the first input field vs. moving it to the top of the visible text above the field (with tabindex = -1). I suppose it depends on if the text is linked to the input field. For example, on the second "tab", the section "Zones" is not linked to the label/input field and could be missed by a screen reader.

But as I said, that level of detail can wait until the a11y review.